website/docs/components/outputs/sql_raw.md

```diff
@@ -152,6 +155,7 @@ The query to execute. The style of placeholder to use depends on the driver, som
 | `postgres` | Dollar sign |
 | `mssql` | Question mark |
 | `sqlite` | Question mark |
+| `oracle` | Colon |
```

Comment:
There seems to be a lot of copy paste overhead between sql files, refering to one example of: output/sql.md:106 output/sql_raw.md:158

Is it possible in markdown to somehow refer to other document's content? It's going to be, in the future, really hard to maintain the documentation.

Reply:
I agree, there is some duplication here that should be eliminated. PRs are welcome :)

However, these .md files are generated automatically via `make docs`, so that should be fine. This bit in particular comes from here: https://github.com/benthosdev/benthos/blob/f0e07b3b3bdb677ba105289c2a4e91753ebf0d38/internal/impl/sql/conn_fields.go#L22-L28